clutch/src/lib.rs

```diff
@@ -102,7 +102,9 @@ impl<F: LurkField> ClutchState<F> {
 }
 impl ReplTrait<F> for ClutchState<F> {
     fn new(s: &mut Store<F>, limit: usize, command: Option<Command>) -> Self {
-        let proof_map = fcomm::nova_proof_cache();
+        let reduction_count = 100;
```

Comment:
It would be good to have this as a file constant (if not a constructor parameter), in order to avoid the magic number in code.